README.md

```diff
@@ -285,7 +339,7 @@ func createTestDatabase() *memory.Database {
 Then, you can connect to the server with any MySQL client:
 
 ```bash
-> mysql --host=127.0.0.1 --port=3306 -u root mydb -e "SELECT * FROM mytable"
+> mysql --host=127.0.0.1 --port=3306 -u user -ppass test -e "SELECT * FROM mytable"
```

Comment:
Seems like you are working on an earlier base copy of README

Make sure you merge main before you regen the README again